playground/buddike/lab1/file_processor.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+
+def readfilecontent(path):
+    print("Reading file content from " + path)
+
+    if os.path.exists(path) == 0:
```

Comment:
python has the same kind of truthy behavior as javascript so 0, false, -1, None are all considered false so as an alternative you can use  
```suggestion
    if not os.path.exists(path):
```